.github/workflows/cloud-dev-build-deploy.yml

```
@@ -0,0 +1,108 @@
+    branches:
+      - develop
+  workflow_dispatch:
+
+jobs:
+  build-docker-image:
+    runs-on: ubuntu-latest
+    environment: dev
+
+    steps:
+      - name: get repo from github
+        uses: actions/checkout@v3
```

actionlint (1.7.9)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)